.eslintrc

```diff
@@ -46,6 +48,9 @@ rules:
   no-use-before-define: 0
   # We do this in a few places to align values
   key-spacing: 0
+  # https://github.com/facebook/react/pull/3968
```

Comment:
Nit: can you actually mention why it's disabled and not just link to a PR which has a lot of unrelated discussion.